Header header
string type
string position
MotorPosition[] motor_positions
